content/en/news/releases/1.6.x/announcing-1.6.x/upgrade-notes/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Upgrade Notes
+description: Important Changes to consider when upgrading to Istio 1.6.
+weight: 20
+---
+
+This page describes changes you need to be aware of when upgrading from Istio
+1.5.x to Istio 1.6.x. Here, we detail cases where we intentionally broke backwards
+compatibility. We also mention cases where backwards compatibility was preserved
+but new behavior was introduced that would be surprising to someone familiar with
+the use and operation of Istio 1.5.
+
+{{< tip >}}
+Istio does not currently support skip-level upgrades. For example, if you are still using Istio 1.4, we recommend first upgrading to Istio 1.5. However, if you choose to upgrade from previous version, you must first disable Galley configuration validation. This can be done by adding `--enable-validation=false` to the Galley deployment and removing the `istio-galley` `ValidatingWebhookConfiguration`
+{{< /tip >}}
+
+## Removal of legacy Helm charts
+
+In Istio 1.4 we introduced a [new way to install Istio](/blog/2019/introducing-istio-operator/), using the in-cluster Operator or `istioctl install` command. As part of this effort, we deprecated the old Helm charts. Over time, we implemented many of the new Istio features only in these new installation methods. As a result, we have decided to remove the old installation Helm charts in Istio 1.6.
+
+We recommend reviewing the [Istio 1.5 Upgrade Notes](/news/releases/1.5.x/announcing-1.5/upgrade-notes/#control-plane-restructuring) before continuing, because we introduced several changes in Istio 1.5 that were not present in the legacy installation method, such as Istiod and Telemetry V2.
```

Comment:
Capitalization? I've seen them both ways. Consistency would be nice.

Reply:
@rcaballeromx Should Telemetry V2 be capitalized?

Reply:
No, `telemetry v2` would be better.

Reply:
Thanks. Fixed.